test/python/fe_api/grouped_gemm/test_grouped_gemm_glu_hadamard_quant.py

```
@@ -0,0 +1,676 @@
+# =============================================================================
+# Reference implementations
+# =============================================================================
+
+
+def _run_grouped_gemm_glu_ref(
+    inputs: Dict,
+    act_func: str,
+    glu_alpha: Optional[float] = None,
+    glu_limit: Optional[float] = None,
+) -> Dict:
+    n, _, l = inputs["b_ref"].shape
```

Ruff (0.16.1)

[error] Ambiguous variable name: `l`

(E741)